main.py

```diff
@@ -15,18 +17,33 @@
 
 initMsgId = 0
 raidpackage_client: RaidPackageClient = None
+error_handler_client: Error_handler = None
+item_handler_client: Item_handler = None
+emoji_service_client: Emoji_service = None
 
 @client.event
 async def on_ready():
     global raidpackage_client
-    populate_items()
-    
+    global error_handler_client
+    global emoji_service_client
+    global item_handler_client
     guild = client.guilds[0]
     confirmed_channel, order_channel = await intialize_order_channels(guild, client, CONFIRMED_ORDER_CHANNEL_ID, ORDER_INIT_CHANNEL_ID)
 
+    if error_handler_client is None:
+        error_handler_client = Error_handler()
+
+    if emoji_service_client is None:
+        emoji_service_client = Emoji_service(guild)
+
+    if item_handler_client is None:
+        item_handler_client = Item_handler(emoji_service_client)
```

Comment:
Items are part of RaidOrder domain so maybe move the initialization to raidpackage all together

Reply:
I am reluctant to do this, in case in the future we want other services to make use of items, such as the trade hub for putting together price packages, or if/when we integrate with an inventory database/spreadsheet for example

Reply:
Fair enough